client/components/App.jsx

```diff
@@ -1,18 +1,85 @@
 import React from 'react';
+import styled from 'styled-components';
+import PriceAndPromoContainer from "./PriceAndPromoContainer.jsx";
+import PriceImage from "./PriceImage.jsx"
+import axios from 'axios';
+
+const AppWrapper = styled.div`
+  width: 370.5px;
+  height: 431.54px;
+`;
+const AppStyled = styled.div`
+  background-color: #1b1e1b;
+`;
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = null;
+  }
+
+  fetchImage(productId) {
+    return "url1";
+  }
+
+  fetchProductPlatform(productId) {
+    return { os: ["urlLinux", "urlWindows"] };
```

Comment:
there is the endpoint for platforms, but os is not included in the results, Platforms are like Steam, Origin etc.  Os is available from main endpoint, unless you would like for there to be os included in the platforms endpoint.  I might just do that anyway shouldn't be difficult.  Let me know

Reply:
omg, would you? That would be so great, Chris. If you're sure it's not too much trouble.